tutorials/3-run-node/index.md

```diff
@@ -468,7 +468,7 @@ $ ./build/simd add-genesis-account alice 100000000stake \
 $ docker run --rm -it \
     -v $(pwd)/private:/root \
     simd:v0.45.4 \
-    simd add-genesis-account alice 100000000stake \
+    simd genesis add-genesis-account alice 100000000stake \
```

Comment:
What version of Cosmos SDK did you compile to get this?

Reply:
this will be from the latest vrsion. it seems these tutorials are of versions not supported by the cosmos sdk any longer. will they be updated?

Reply:
Also add genesis at lines 458, 504, 518, 540, 552.